src/3.2/docker-entrypoint.sh

```diff
@@ -1,30 +1,77 @@
+    NEO4J_dbms_tx__log_rotation_retention_policy=${NEO4J_dbms_txLog_rotation_retentionPolicy:-}
+    NEO4J_dbms_memory_pagecache_size=${NEO4J_dbms_memory_pagecache_size:-}
+    NEO4J_wrapper_java_additional=${NEO4J_UDC_SOURCE:-}
+    NEO4J_dbms_memory_heap_initial__size=${NEO4J_dbms_memory_heap_maxSize:-}
+    NEO4J_dbms_memory_heap_max__size=${NEO4J_dbms_memory_heap_maxSize:-}
+    NEO4J_dbms_unmanaged__extension__classes=${NEO4J_dbms_unmanagedExtensionClasses:-}
+    NEO4J_dbms_allow__format__migration=${NEO4J_dbms_allowFormatMigration:-}
+    NEO4J_dbms_mode=${NEO4J_dbms_mode:-}
+    NEO4J_dbms_connectors_default__advertised__address=${NEO4J_dbms_connectors_defaultAdvertisedAddress:-}
+    NEO4J_ha_server__id=${NEO4J_ha_serverId:-}
+    NEO4J_ha_host_data=${NEO4J_ha_host_data:-}
+    NEO4J_ha_host_coordination=${NEO4J_ha_host_coordination:-}
+    NEO4J_ha_initial__hosts=${NEO4J_ha_initialHosts:-}
+    NEO4J_causal__clustering_expected__core__cluster__size=${NEO4J_causalClustering_expectedCoreClusterSize:-}
+    NEO4J_causal__clustering_initial__discovery__members=${NEO4J_causalClustering_initialDiscoveryMembers:-}
+    NEO4J_causal__clustering_discovery__listen__address=${NEO4J_causalClustering_discoveryListenAddress:-}
+    NEO4J_causal__clustering_discovery__advertised__address=${NEO4J_causalClustering_discoveryAdvertisedAddress:-}
+    NEO4J_causal__clustering_transaction__listen__address=${NEO4J_causalClustering_transactionListenAddress:-}
+    NEO4J_causal__clustering_transaction__advertised__address=${NEO4J_causalClustering_transactionAdvertisedAddress:-}
+    NEO4J_causal__clustering_raft__listen__address=${NEO4J_causalClustering_raftListenAddress:-}
+    NEO4J_causal__clustering_raft__advertised__address=${NEO4J_causalClustering_raftAdvertisedAddress:-}
+
+    # Custom settings for dockerized neo4j
+    NEO4J_dbms_tx__log_rotation_retention_policy=${NEO4J_dbms_tx__log_rotation_retention_policy:-100M size}
+    NEO4J_dbms_memory_pagecache_size=${NEO4J_dbms_memory_pagecache_size:-512M}
+    NEO4J_wrapper_java_additional=${NEO4J_wrapper_java_additional:--Dneo4j.ext.udc.source=docker}
+    NEO4J_dbms_memory_heap_initial__size=${NEO4J_dbms_memory_heap_initial__size:-512M}
+    NEO4J_dbms_memory_heap_max__size=${NEO4J_dbms_memory_heap_max__size:-512M}
+    NEO4J_dbms_connectors_default__listen__address=${NEO4J_dbms_connectors_default__listen__address:-0.0.0.0}
+    NEO4J_dbms_connector_http_listen__address=${NEO4J_dbms_connector_http_listen__address:-0.0.0.0:7474}
+    NEO4J_dbms_connector_https_listen__address=${NEO4J_dbms_connector_https_listen__address:-0.0.0.0:7473}
+    NEO4J_dbms_connector_bolt_listen__address=${NEO4J_dbms_connector_bolt_listen__address:-0.0.0.0:7687}
+    NEO4J_causal__clustering_discovery__listen__address=${NEO4J_causal__clustering_discovery__listen__address:-0.0.0.0:5000}
+    NEO4J_causal__clustering_discovery__advertised__address=${NEO4J_causal__clustering_discovery__advertised__address:-$(hostname):5000}"
```

Comment:
Inconsistent use of quotation marks here. The rest of the file becomes quoted and fails a syntax check